.changeset/chatty-windows-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-api-docs-backend': major
```

Comment:
```suggestion
'@backstage/plugin-api-docs-backend': minor
```

Package starts out at `0.0.0` and then we bump it up to `0.1.0` for the first release.